tests/autoyast/installation.pm

```diff
@@ -127,7 +128,7 @@ sub run {
     push @needles, 'autoyast-error' unless get_var('AUTOYAST_EXPECT_ERRORS');
     # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
     # So push a needle to check upcoming reboot on zVM that is a way to indicate the stage done
-    push @needles, 'autoyast-stage1-reboot-upcoming' if is_s390x || (is_pvm && !is_upgrade);
+    push @needles, 'autoyast-stage1-reboot-upcoming' if is_s390x || (is_pvm && !is_upgrade) || (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'hvm'));
```

Comment:
Is it fine for setting `check_var('VIRSH_VMM_TYPE', 'hvm')` only？

Reply:
For me with these two setting together to make the condition more accuracy for xen-hvm, so I prefer to use these two together as condition. Thanks.